Gems/PhysX/Code/Editor/Source/Components/EditorSystemComponent.cpp

```diff
@@ -236,9 +244,9 @@ namespace PhysX
             if (!resultAssetId.IsValid())
             {
                 // No file for the default material library, create it
-                const char* assetRoot = AZ::IO::FileIOBase::GetInstance()->GetAlias("@projectsourceassets@");
+                const char* projectRoot = AZ::IO::FileIOBase::GetInstance()->GetAlias("@projectroot@");
```

Comment:
This wouldn't work because `DefaultAssetFilePath` does need to include "Assets/" as well, in other places it's used to call `GetAssetIdByPath` which needs "Assets/" too. 

Apart from that, are you also suggesting to use `FilePathKey_ProjectPath` instead of `AZ::IO::FileIOBase::GetInstance()->GetAlias`?

Reply:
I look at you PR, I'll change it to use settingsRegistry (same as your code but  without the `fullPath /= "Assets";` line because it's already part of `DefaultAssetFilePath`).

Reply:
I can make that change in PR then.

Reply:
Actually, I'll let you make that change.

Reply:
I've already done it here :)